xml/System.ComponentModel/DateOnlyConverter.xml

```diff
@@ -57,7 +57,7 @@
       <Docs>
         <param name="context">To be added.</param>
         <param name="sourceType">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Gets a value indicating whether this converter can convert an object in the given source type to a <see cref="T:System.DateOnly" /> object using the specified context.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion        
        <returns><see langword="true" /> if this object can perform the conversion; otherwise, <see langword="false" />.</returns>
```